### DISCUSSION TITLE
建议用国内商品期货的交易数据来训练

### DISCUSSION THREAD

**Discussion by lihongteng** [AI4Finance-Foundation/FinGPT#58] [Ideas]
我是国内商品期货从业者，和交易者，能够稳定盈利的交易者朋友资源比较多。国内期货交易数据是可获得的，对于量化交易来说，训练成交数据就足够得住交易策略了。我想做出我的贡献，如果有作用的话。
I am a domestic commodity futures practitioner and trader, and there are more friends who can make stable profits. Domestic futures trading data is available, and for quantitative trading, training trading data is sufficient to justify trading strategies. I want to make my contribution, if it makes a difference.

建议用中国商品期货交易数据来训练的原因是交易成本更低，成熟的量化交易平台有很多，调用历史交易数据网络上也可以找到。我认为这个是可行的，如果有作用，我可以提供任何帮助。

**Comment by YangletLiu** [comment]
> 我是国内商品期货从业者，和交易者，能够稳定盈利的交易者朋友资源比较多。国内期货交易数据是可获得的，对于量化交易来说，训练成交数据就足够得住交易策略了。我想做出我的贡献，如果有作用的话。 I am a domestic commodity futures practitioner and trader, and there are more friends who can make stable profits. Domestic futures trading data is available, and for quantitative trading, training trading data is sufficient to justify trading strategies. I want to make my contribution, if it makes a difference.
> 
> 建议用中国商品期货交易数据来训练的原因是交易成本更低，成熟的量化交易平台有很多，调用历史交易数据网络上也可以找到。我认为这个是可行的，如果有作用，我可以提供任何帮助。

Thanks very much! It is a great idea, looking forward to your PRs if you have time to play with it.

**Reply by lihongteng** [reply]
首先，我不懂编程，我只知道哪个交易平台比较领先，知道数据在哪里。训练模型最直接的就是去训练历史的交易数据。并且把它作为一个奖惩游戏让机器人得出交易策略。机器需要选择下注比例和下注涨跌，通过有监督的方式去不断修正他的模型，得出合理策略。

First of all, I don't know programming, I just know which trading platform is ahead and know where the data is. The most straightforward way to train a model is to train historical transaction data. And use it as a reward and punishment game for the robot to come up with a trading strategy. The machine needs to choose the proportion of bets and the ups and downs of bets, and constantly revise its model in a supervised way to arrive at a reasonable strategy.

**Comment by lihongteng** [comment]
> Thanks very much! It is a great idea, looking forward to your PRs if you have time to play with it.

首先，历史交易数据就足够机器人分析出市场参与者的心理情绪，而且是客观数据。如果通过市场的消息去分析市场的走势，那将很难有稳定的模型策略，因为市场的消息本身就是携带主观意图，有些消息是为了掩饰真相。通过有监督的机制去训练模型，才是出路。
First of all, historical trading data is enough for robots to analyze the psychological emotions of market participants, and it is objective data. If you analyze the market trend through the market news, it will be difficult to have a stable model strategy, because the market news itself is carrying subjective intentions, and some news is to hide the truth. Training models through supervised mechanisms is the way forward.
